packages/angular_devkit/build_angular/src/utils/process-bundle.ts

```diff
@@ -693,10 +693,28 @@ async function inlineLocalesDirect(ast: ParseResult, options: InlineOptions) {
   }
 
   const { default: generate } = await import('@babel/generator');
-  const utils = await import(
-    // tslint:disable-next-line: trailing-comma no-implicit-dependencies
-    '@angular/localize/src/tools/src/translate/source_files/source_file_utils'
-  );
+
+  // Remove the below try/catch before v10 RC
+  // tslint:disable-next-line: trailing-comma no-implicit-dependencies
+  let utils: typeof import('@angular/localize/src/tools/src/translate/source_files/source_file_utils');
```

Comment:
I believe that this line will fail when you update to latest master too...

Reply:
That will fail only if we bump the dependencies in the repo. This will need to be done after the FW cuts a release.